Form/Type/DateRangeType.php

```diff
@@ -20,15 +20,24 @@
 class DateRangeType extends AbstractType
 {
     /**
-     * @var TranslatorInterface
+     * @var TranslatorInterface|null
+     *
+     * @deprecated Translator property is deprecated since version 3.1, to be removed in 4.0.
      */
     protected $translator;
 
     /**
-     * @param TranslatorInterface $translator
+     * @param TranslatorInterface|null $translator
+     *
+     * @deprecated Translator dependency is deprecated since version 3.1, to be removed in 4.0.
```

Comment:
We use this notation, because we can't guarantee what the next version will be or rather, when this PR will be merged.